<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>764</width>
    <height>460</height>
   </rect>
  </property>
  <property name="minimumSize">
   <size>
    <width>764</width>
    <height>460</height>
   </size>
  </property>
  <property name="maximumSize">
   <size>
    <width>764</width>
    <height>460</height>
   </size>
  </property>
  <property name="windowTitle">
   <string>Youtube Downloader</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <widget class="QTabWidget" name="tabWidget">
    <property name="enabled">
     <bool>true</bool>
    </property>
    <property name="geometry">
     <rect>
      <x>-10</x>
      <y>0</y>
      <width>781</width>
      <height>481</height>
     </rect>
    </property>
    <property name="minimumSize">
     <size>
      <width>781</width>
      <height>481</height>
     </size>
    </property>
    <property name="maximumSize">
     <size>
      <width>781</width>
      <height>481</height>
     </size>
    </property>
    <property name="toolTip">
     <string>&lt;html&gt;&lt;head/&gt;&lt;body&gt;&lt;p&gt;dsa&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
    </property>
    <property name="styleSheet">
     <string notr="true">background-color:rgb(28, 28, 28);
color: rgb(181, 181, 181);</string>
    </property>
    <property name="currentIndex">
     <number>0</number>
    </property>
    <widget class="QWidget" name="tab">
     <property name="palette">
      <palette>
       <active>
        <colorrole role="WindowText">
         <brush brushstyle="SolidPattern">
          <color alpha="255">
           <red>181</red>
           <green>181</green>
           <blue>181</blue>
          </color>
         </brush>
        </colorrole>
        <colorrole role="Button">
         <brush brushstyle="SolidPattern">
          <color alpha="255">
           <red>28</red>
           <green>28</green>
           <blue>28</blue>
          </color>
         </brush>
        </colorrole>
        <colorrole role="Text">
         <brush brushstyle="SolidPattern">
          <color alpha="255">
           <red>181</red>
           <green>181</green>
           <blue>181</blue>
          </color>
         </brush>
        </colorrole>
        <colorrole role="ButtonText">
         <brush brushstyle="SolidPattern">
          <color alpha="255">
           <red>181</red>
           <green>181</green>
           <blue>181</blue>
          </color>
         </brush>
        </colorrole>
        <colorrole role="Base">
         <brush brushstyle="SolidPattern">
          <color alpha="255">
           <red>28</red>
           <green>28</green>
           <blue>28</blue>
          </color>
         </brush>
        </colorrole>
        <colorrole role="Window">
         <brush brushstyle="SolidPattern">
          <color alpha="255">
           <red>28</red>
           <green>28</green>
           <blue>28</blue>
          </color>
         </brush>
        </colorrole>
        <colorrole role="HighlightedText">
         <brush brushstyle="SolidPattern">
          <color alpha="255">
           <red>0</red>
           <green>0</green>
           <blue>0</blue>
          </color>
         </brush>
        </colorrole>
        <colorrole role="ToolTipBase">
         <brush brushstyle="SolidPattern">
          <color alpha="255">
           <red>0</red>
           <green>0</green>
           <blue>0</blue>
          </color>
         </brush>
        </colorrole>
       </active>
       <inactive>
        <colorrole role="WindowText">
         <brush brushstyle="SolidPattern">
          <color alpha="255">
           <red>181</red>
           <green>181</green>
           <blue>181</blue>
          </color>
         </brush>
        </colorrole>
        <colorrole role="Button">
         <brush brushstyle="SolidPattern">
          <color alpha="255">
           <red>28</red>
           <green>28</green>
           <blue>28</blue>
          </color>
         </brush>
        </colorrole>
        <colorrole role="Text">
         <brush brushstyle="SolidPattern">
          <color alpha="255">
           <red>181</red>
           <green>181</green>
           <blue>181</blue>
          </color>
         </brush>
        </colorrole>
        <colorrole role="ButtonText">
         <brush brushstyle="SolidPattern">
          <color alpha="255">
           <red>181</red>
           <green>181</green>
           <blue>181</blue>
          </color>
         </brush>
        </colorrole>
        <colorrole role="Base">
         <brush brushstyle="SolidPattern">
          <color alpha="255">
           <red>28</red>
           <green>28</green>
           <blue>28</blue>
          </color>
         </brush>
        </colorrole>
        <colorrole role="Window">
         <brush brushstyle="SolidPattern">
          <color alpha="255">
           <red>28</red>
           <green>28</green>
           <blue>28</blue>
          </color>
         </brush>
        </colorrole>
        <colorrole role="HighlightedText">
         <brush brushstyle="SolidPattern">
          <color alpha="255">
           <red>0</red>
           <green>0</green>
           <blue>0</blue>
          </color>
         </brush>
        </colorrole>
        <colorrole role="ToolTipBase">
         <brush brushstyle="SolidPattern">
          <color alpha="255">
           <red>0</red>
           <green>0</green>
           <blue>0</blue>
          </color>
         </brush>
        </colorrole>
       </inactive>
       <disabled>
        <colorrole role="WindowText">
         <brush brushstyle="SolidPattern">
          <color alpha="255">
           <red>181</red>
           <green>181</green>
           <blue>181</blue>
          </color>
         </brush>
        </colorrole>
        <colorrole role="Button">
         <brush brushstyle="SolidPattern">
          <color alpha="255">
           <red>28</red>
           <green>28</green>
           <blue>28</blue>
          </color>
         </brush>
        </colorrole>
        <colorrole role="Text">
         <brush brushstyle="SolidPattern">
          <color alpha="255">
           <red>181</red>
           <green>181</green>
           <blue>181</blue>
          </color>
         </brush>
        </colorrole>
        <colorrole role="ButtonText">
         <brush brushstyle="SolidPattern">
          <color alpha="255">
           <red>181</red>
           <green>181</green>
           <blue>181</blue>
          </color>
         </brush>
        </colorrole>
        <colorrole role="Base">
         <brush brushstyle="SolidPattern">
          <color alpha="255">
           <red>28</red>
           <green>28</green>
           <blue>28</blue>
          </color>
         </brush>
        </colorrole>
        <colorrole role="Window">
         <brush brushstyle="SolidPattern">
          <color alpha="255">
           <red>28</red>
           <green>28</green>
           <blue>28</blue>
          </color>
         </brush>
        </colorrole>
        <colorrole role="HighlightedText">
         <brush brushstyle="SolidPattern">
          <color alpha="255">
           <red>0</red>
           <green>0</green>
           <blue>0</blue>
          </color>
         </brush>
        </colorrole>
        <colorrole role="ToolTipBase">
         <brush brushstyle="SolidPattern">
          <color alpha="255">
           <red>0</red>
           <green>0</green>
           <blue>0</blue>
          </color>
         </brush>
        </colorrole>
       </disabled>
      </palette>
     </property>
     <attribute name="title">
      <string>                                                                                                                       Download                                                                                                                    </string>
     </attribute>
     <widget class="QPushButton" name="pushButton">
      <property name="geometry">
       <rect>
        <x>590</x>
        <y>30</y>
        <width>161</width>
        <height>81</height>
       </rect>
      </property>
      <property name="styleSheet">
       <string notr="true">background-color: rgb(51, 51, 51);
border-radius: 10px;
color: rgb(181, 181, 181);
font-size: 18px</string>
      </property>
      <property name="text">
       <string>Скачать</string>
      </property>
     </widget>
     <widget class="QLineEdit" name="lineEdit">
      <property name="geometry">
       <rect>
        <x>22</x>
        <y>30</y>
        <width>561</width>
        <height>81</height>
       </rect>
      </property>
      <property name="styleSheet">
       <string notr="true">background-color: rgb(181, 181, 181);
border-radius: 10px;
color: rgb(56, 56, 56);
font-size: 18px</string>
      </property>
      <property name="text">
       <string/>
      </property>
      <property name="placeholderText">
       <string>    Введите ссылку...</string>
      </property>
     </widget>
     <widget class="QLabel" name="label">
      <property name="geometry">
       <rect>
        <x>20</x>
        <y>160</y>
        <width>141</width>
        <height>51</height>
       </rect>
      </property>
      <property name="styleSheet">
       <string notr="true">background-color: rgb(51, 51, 51);
border-radius: 10px;
color: rgb(181, 181, 181);
font-size: 18px</string>
      </property>
      <property name="text">
       <string> Длительность:</string>
      </property>
     </widget>
     <widget class="QLabel" name="label_2">
      <property name="geometry">
       <rect>
        <x>20</x>
        <y>260</y>
        <width>141</width>
        <height>51</height>
       </rect>
      </property>
      <property name="styleSheet">
       <string notr="true">background-color: rgb(51, 51, 51);
border-radius: 10px;
color: rgb(181, 181, 181);
font-size: 18px</string>
      </property>
      <property name="text">
       <string>      Размер  :</string>
      </property>
     </widget>
     <widget class="QLabel" name="label_3">
      <property name="geometry">
       <rect>
        <x>20</x>
        <y>360</y>
        <width>141</width>
        <height>51</height>
       </rect>
      </property>
      <property name="styleSheet">
       <string notr="true">background-color: rgb(51, 51, 51);
border-radius: 10px;
color: rgb(181, 181, 181);
font-size: 18px</string>
      </property>
      <property name="text">
       <string>  Разрешение :</string>
      </property>
     </widget>
     <widget class="QComboBox" name="comboBox">
      <property name="geometry">
       <rect>
        <x>180</x>
        <y>360</y>
        <width>581</width>
        <height>51</height>
       </rect>
      </property>
      <property name="font">
       <font>
        <pointsize>-1</pointsize>
        <weight>50</weight>
        <italic>false</italic>
        <bold>false</bold>
        <kerning>true</kerning>
       </font>
      </property>
      <property name="styleSheet">
       <string notr="true">background-color: rgb(51, 51, 51);
border-radius: 10px;
color: rgb(181, 181, 181);
font-size: 18px</string>
      </property>
      <property name="editable">
       <bool>false</bool>
      </property>
     </widget>
     <widget class="QLineEdit" name="lineEdit_3">
      <property name="geometry">
       <rect>
        <x>180</x>
        <y>160</y>
        <width>581</width>
        <height>51</height>
       </rect>
      </property>
      <property name="styleSheet">
       <string notr="true">background-color: rgb(51, 51, 51);
border-radius: 10px;
color: rgb(181, 181, 181);
font-size: 18px</string>
      </property>
     </widget>
     <widget class="QLineEdit" name="lineEdit_4">
      <property name="geometry">
       <rect>
        <x>180</x>
        <y>260</y>
        <width>581</width>
        <height>51</height>
       </rect>
      </property>
      <property name="styleSheet">
       <string notr="true">background-color: rgb(51, 51, 51);
border-radius: 10px;
color: rgb(181, 181, 181);
font-size: 18px</string>
      </property>
     </widget>
    </widget>
   </widget>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
